### DISCUSSION TITLE
Cannot find module `@atlastkit/..`. or its corresponding type declarations.

### DISCUSSION THREAD

**Discussion by william-low1031** [atlassian/pragmatic-drag-and-drop#56] [Q&A]
Hi I have a question about using @atlastkit drag and drop. I am trying to understand what is going wrong and wanna know any possible way to solve it, sorry if I asking bad question.

I tried to import the package but I don't have any import error problem but having this error when I am trying to start the project. I saw there is a issue been opened for the `exports` but I am not sure whether they are related. 
https://github.com/atlassian/pragmatic-drag-and-drop/issues/27

I have been using version 1.1.9

![Screenshot 2024-05-23 at 10 27 27](https://github.com/atlassian/pragmatic-drag-and-drop/assets/56996459/baa2c6d6-f712-4e55-9eca-44d1a90a79b9)

Thanks for reading this far and have a nice day. Extra thank you for helping out.

**Comment by SiteAppCreators** [comment]
Have you tried uninstalling and reinstalling the pragmatic drag and drop package?
Try installing with "npm install @atlaskit/pragmatic-drag-and-drop@latest" 
The latest version is now 1.1.12

**Reply by benjosua** [reply]
I have the same error and simply doing "pnpm install @atlaskit/pragmatic-drag-and-drop@latest" doesn't work. Any ideas?

**Reply by ronaldgj** [reply]
Also got the issue with `"@atlaskit/pragmatic-drag-and-drop": "^1.2.1"`.
![Scherm­afbeelding 2024-07-26 om 16 54 58](https://github.com/user-attachments/assets/5b77611c-2410-4dc4-86f7-a5768cd56973)
